scripts/fuzz_opt.py

```diff
@@ -1535,7 +1544,8 @@ def randomize_opt_flags():
               '(mean:', str(mean) + ', stddev:', str(stddev) + ')',
               'speed:', counter / elapsed,
               'iters/sec, ', total_wasm_size / elapsed,
-              'wasm_bytes/sec\n')
+              'wasm_bytes/sec, ', ignored_vm_runs,
+              'ignored\n')
```

Comment:
It's a little fuzzy actually since we can do more than one VM run in a single fuzzer iteration. For example we might run CompareVMs and also FuzzExec, and end up running wasm-opt twice, and so we might get two ignores for a single run. Maybe it's worth trying to be more precise here, I'm not sure, but anyhow the actual rate isn't available without some more work.

Reply:
(That 1% I mentioned is an upper limit, in other words, on the number of iterations that have at least one ignored VM run inside them.)